.github/workflows/pykanidm_pytest.yml

```diff
@@ -0,0 +1,28 @@
+name: pykanidm - pytest
+
+"on":
+  push:
+  pull_request:
+jobs:
+  pykanidm_pytest:
+    strategy:
+      matrix:
+        python_version:
+          - "3.8"
```

Comment:
Should we define a min-python somewhere?

Reply:
It's defined in the `pyproject.toml`'s as 3.8, could add that to the support docs above if that'd help?

Reply:
Nah, lets leave it in one place only.